cmd/snap/cmd_run.go

```diff
@@ -33,10 +34,12 @@ import (
 
 type cmdRun struct {
 	Positional struct {
-		SnapApp string `positional-arg-name:"<app name>" description:"the application to run, e.g. hello-world.env"`
+		SnapApp string `positional-arg-name:"<app name>" description:"the snap (e.g. hello-world) or application to run (e.g. hello-world.env)"`
 	} `positional-args:"yes" required:"yes"`
 
-	Command string `long:"command" description:"alternative command to run"`
+	Command  string `long:"command" description:"alternative command to run"`
+	Hook     string `long:"hook" description:"run a specific hook contained within the snap"`
```

Comment:
We can use just `"hook to run"` here. Similar to the rationale of the description for --command, it's okay that people don't understand details. Realistically this should only be used by people that know what they are doing.

In fact, is it easy to hide both of these options?  Does "hidden" work here too? I'd prefer that over showing them in the help.